server/controllers/medical/patientGroups.js

```diff
@@ -30,8 +30,15 @@ function list(req, res, next) {
   const filters = new FilterParser(req.query, { tableAlias : 'pg' });
   const sql = `
     SELECT BUID(pg.uuid) as uuid, pg.name, BUID(pg.price_list_uuid) AS price_list_uuid,
-      pg.note, pg.created_at, pl.label AS priceListLabel, pl.description
-    FROM patient_group AS pg LEFT JOIN price_list AS pl ON pg.price_list_uuid = pl.uuid
+      pg.note, pg.created_at, pl.label AS priceListLabel, pl.description,
```

Comment:
Is this needed for the Price List group count...?

Reply:
It's used in the patient group grid, we also need to know how many patients belong to a group